### PR TITLE
Update language support

### PR DESCRIPTION
**Pull request by react-learner**
Fix inaccurate language support section (we do not support Japanese + Chinese) + improve explanation of contribution options

These changes come out of a conversation in the weekly docs sync (notes will be posted online at a later point).

### CODE REVIEW

File: guides/advanced_guides/tokenization.md [hunk 1]
```diff
@@ -21,24 +21,26 @@ We can break down the tokenization process like so:
...
 
 ## Future Development
 
-We rely on external contributors to help us extend our tokenizer to support new languages.
+While we have employees from all over the world at MeiliSearch, we don't speak every language. In fact, we rely almost entirely on external contributors to understand how our engine is performing in a given language.
 
-In the future, we hope that our international developer community will lead the process of developing additional tokenizer pipelines. For now, if there's an unsupported language that you would like us to target, you can help by writing tests for that language, giving us feedback, or simply giving a :+1: to the relevant GitHub issue.
+In the future, we plan to open-source the development of [tokenization pipelines](/guides/advanced_guides/tokenization.md#tokenization) and let our international community lead the charge towards a more global MeiliSearch. For now, you can help by sharing tests, results, and feedback with us in that language's GitHub issue. Be sure to also give a :+1: to any language you'd like us to target!
```
Comment (bidoubiwa) [inline guides/advanced_guides/tokenization.md:37]: I'm not sure about this. @MarinPostma could you confirm?

**Review by bidoubiwa — COMMENTED** [review]
great changes!

File: guides/advanced_guides/tokenization.md (hunk 1 above)
Comment (MarinPostma) [reply guides/advanced_guides/tokenization.md:37]: There are no officials plan for the tokenizer, it's a long term effort, so d rather we do not commit on anything. You can point to the repo for now, so curious people can have a look and contribute is they want to.

Comment (MarinPostma) [reply guides/advanced_guides/tokenization.md:37]: The tokenizer is open source, but the development is lead by us still 😊

File: guides/advanced_guides/tokenization.md [hunk 2]
```diff
@@ -21,24 +21,26 @@ We can break down the tokenization process like so:
...
+Currently, MeiliSearch offers full support for two groups of languages:
 
-- **English**  🇬🇧
+- **English** 🇬🇧 and **any other language that uses the Latin alphabet**
 - **Chinese** 🇨🇳
-- **Japanese** 🇯🇵
-- **Korean** 🇰🇷
 
-It also supports any language that uses the Latin alphabet. **Results may vary in languages with long compound words, such as German.**
+This is due to how our [tokenizer](/guides/advanced_guides/tokenization.md#tokenization) is optimized. While we can't promise high-quality results for languages outside of these two groups, we find that **MeiliSearch works effectively in most languages**.
```
Comment (MarinPostma) [inline guides/advanced_guides/tokenization.md:29]: I'm not comfortable with this section, it make it sound like meili may not work well on most languages. I would turn it to our advantage and say that we have multi-lingual support, and that chinese is supported through jieba.

It's better to have user come to us with concrete error than people that get turned down from trying meili so we don't know how to improve their experience

**Review by MarinPostma — CHANGES_REQUESTED** [review]

**Review by bidoubiwa — DISMISSED** [review]
👌

File: guides/advanced_guides/tokenization.md [hunk 3]
```diff
@@ -17,28 +17,39 @@ When you add documents to a MeiliSearch index, the tokenization process is handl
 We can break down the tokenization process like so:
 
 1. Crawl the document(s) and determine the primary language for each field.
-2. Go back over the documents field-by-field, running the corresponding tokenization pipeline, if it exists. Pipelines include many language-specific operations.
+2. Go back over the documents field-by-field, running the corresponding tokenization pipeline, if it exists.
+
+Pipelines include many language-specific operations. Currently, we have two pipelines:
+
+1. A specialized Chinese pipeline using [Jieba](https://github.com/messense/jieba-rs)
+2. A default pipeline that uses [unicode text segmentation](https://github.com/unicode-rs/unicode-segmentation).
```
Comment (MarinPostma) [inline guides/advanced_guides/tokenization.md:25]: We don't use unicode segmenter yet, for now this is the same tokenizer as before, that basically separate words based on categories (separators and words components) to put things shortly.

File: guides/advanced_guides/tokenization.md [hunk 4]
```diff
@@ -17,28 +17,39 @@ When you add documents to a MeiliSearch index, the tokenization process is handl
...
+Don't know if your language is covered? **We recommend trying MeiliSearch anyway.** MeiliSearch is a flexible engine designed to deliver results in any language. We are constantly working to improve our multilingual support and [welcome your feedback](/guides/advanced_guides/tokenization.md#future-development).
+
+If you'd like more information on how MeiliSearch handles different languages, learn about our [tokenizer](/guides/advanced_guides/tokenization.md#tokenization).
 
 ## Future Development
 
-We rely on external contributors to help us extend our tokenizer to support new languages.
+There are many ways you can contribute to creating a more global MeiliSearch.
+
+If you're a Rust dev, check out our [tokenizer repo](https://github.com/meilisearch/Tokenizer) and consider developing a [tokenization pipeline](/guides/advanced_guides/tokenization.md#tokenization) for your language of choice.
```
Comment (MarinPostma) [inline guides/advanced_guides/tokenization.md:44]: For reasons I have exposed [here](https://github.com/meilisearch/devrel/issues/114#issuecomment-763209031), I'd rather we don't advertise too much on contribution on this repo for now. We don't really have the resources to handle this right now :)

File: guides/advanced_guides/tokenization.md [hunk 5]
```diff
@@ -17,28 +17,39 @@ When you add documents to a MeiliSearch index, the tokenization process is handl
...
 
-- **English**  🇬🇧
-- **Chinese** 🇨🇳
-- **Japanese** 🇯🇵
-- **Korean** 🇰🇷
+- **Any language that uses whitespace to separate words**
+- **Chinese** 🇨🇳 (through [Jieba](https://github.com/messense/jieba-rs))
 
-It also supports any language that uses the Latin alphabet. **Results may vary in languages with long compound words, such as German.**
+Don't know if your language is covered? **We recommend trying MeiliSearch anyway.** MeiliSearch is a flexible engine designed to deliver results in any language. We are constantly working to improve our multilingual support and [welcome your feedback](/guides/advanced_guides/tokenization.md#future-development).
```
Comment (MarinPostma) [inline guides/advanced_guides/tokenization.md:36]: I would skip directly to the "feedback welcome part" with something like:

"We aim at providing global language support, and your feedback helps us improving in that sense. If you notice inconsistencies in the way your documents are processed, please open an issue on our [GitHub Repository](link_here)"

Comment (MarinPostma) [reply guides/advanced_guides/tokenization.md:36]: It really sounds like a trap => "Don't know if your language is covered? **We recommend trying MeiliSearch anyway.**" haha

File: guides/advanced_guides/tokenization.md [hunk 6]
```diff
@@ -17,28 +17,39 @@ When you add documents to a MeiliSearch index, the tokenization process is handl
...
 
-- [Thai](https://github.com/meilisearch/MeiliSearch/issues/864)
-- [Persian/Farsi](https://github.com/meilisearch/MeiliSearch/issues/553)
-- [Lao](https://github.com/meilisearch/MeiliSearch/issues/563)
 - [Arabic](https://github.com/meilisearch/MeiliSearch/issues/554)
+- [Lao](https://github.com/meilisearch/MeiliSearch/issues/563)
+- [Persian/Farsi](https://github.com/meilisearch/MeiliSearch/issues/553)
+- [Thai](https://github.com/meilisearch/MeiliSearch/issues/864)
+
+If you'd like us to add or improve support for a language that isn't in the above list, please create an [issue](https://github.com/meilisearch/MeiliSearch/issues/new?assignees=&labels=&template=feature_request.md&title=) saying so, and then make a [pull request on the documentation](https://github.com/meilisearch/documentation/edit/master/guides/advanced_guides/tokenization.md) to add it to the above list.
```
Comment (MarinPostma) [inline guides/advanced_guides/tokenization.md:55]: :+1:

**Review by MarinPostma — CHANGES_REQUESTED** [review]
modulo what I added, I think this is a good start for the time being. Until we stabilize the tokenizer, and have more time to dedicate to this issue, we don't need more. We'll come back to it at some point and got more in depth like https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-tokenizers.html

File: index.md [hunk 7]
```diff
@@ -35,7 +35,7 @@ Why should you use MeiliSearch instead of any other existing solution? We try to
 - **Instant Search** (answers < 50 milliseconds): Priority is given to fast answers for smooth search experience.
 - **Search as you type** (_prefix search_): Results are updated on each keystroke. To make this possible, we use a [prefix-search](/guides/advanced_guides/prefix.md#prefix-search).
 - [Typo tolerance](/guides/advanced_guides/typotolerance.md#typo-tolerance): Understands typo and miss-spelling.
-- [Tokenization](/guides/advanced_guides/tokenization.md) in **English**, **Chinese**, **Japanese**, **Korean**, and languages that use the Latin alphabet.
+- [Tokenization](/guides/advanced_guides/tokenization.md) in **English**, **Chinese**, and **all languages that separate words with whitespace**.
```
Comment (bidoubiwa) [inline index.md:38]: Apparently the better word for this is `word divider` as per wikipedia: https://en.wikipedia.org/wiki/Word_divider#Types

```suggestion
- [Tokenization](/guides/advanced_guides/tokenization.md) in **English**, **Chinese**, and **all languages that uses space as a word divider**.
```
I'm really not sure of my suggestion so feel free to ignore. In which case LGTM 🔥

**Review by bidoubiwa — DISMISSED** [review]

**Review by bidoubiwa — APPROVED** [review]
LGTM

**Pull request merged**